Community/gsoc-alumni.md

```diff
@@ -184,4 +344,36 @@ The Manuscripts project, which is a part of the Grimoire Toolset, helps us in an
 
 #### Social Media
 
-* [LinkedIn](https://www.linkedin.com/in/pranjalaswani/)
\ No newline at end of file
+* [LinkedIn](https://www.linkedin.com/in/pranjalaswani/)
+
+
+## CHAOSS Outreachy students from May 2020 to August 2020 Interns
+
+|Student|Project|Mentors|
+|-------|:-----:|-------|
+|[Ore-Aruwaji Oloruntola](#Ore-Aruwaji-Oloruntola)|Build Workflow Process for CHAOSS Diversity & Inclusion Badging|Matt Germonprez, Matt Snell, Saleh Abdel Motaal|
+
+
+### Ore-Aruwaji Oloruntola
```

Comment:
I would prefer to have all students in chronological order, not separated out by program. Could you put this section please with the GSoC 2020 students?